npm/qsharp/src/language-service/language-service.ts

```diff
@@ -151,8 +156,14 @@ export class QSharpLanguageService implements ILanguageService {
     documentUri: string,
     version: number,
     code: string,
+    languageId: string,
```

Comment:
I think the samne thing here, if this can be made optional it will help with compatibility.